### DISCUSSION TITLE
how to delete instance?

### DISCUSSION THREAD

**Discussion by ewof** [misskey-dev/misskey#11456] [Q&A]
![image](https://github.com/misskey-dev/misskey/assets/26354814/6c154c7e-d012-43d3-962c-d388ca9b8f65)
i want to delete these, i tried block them but they dont go away

**Comment by mappi-pr** [comment]
@ewof 
Servers that have performed a "Lookup" at least once in the past will appear in the "Federation" menu.
If you don't see the excluded server when you narrow down the "Status" by "Federation", it means that the block has been completed and new notes are not being acquired.
Also, I don't think "Lookup" Notes are automatically deleted.
